cjworkbench/i18n/trans.py

```diff
@@ -45,26 +53,39 @@ def trans(message_id, *, default, parameters={}):
 """
 
 
-def localize(locale_id, message_id, *, default, parameters={}):
-    """Localize the given message ID to the given locale."""
+def localize(
+    locale_id: str, message_id: str, parameters: _Parameters = {}
+) -> Optional[str]:
+    """Localize the given message ID to the given locale.
+
+    If the message is not found in the catalog for the given or the default locale, return `None`.
```

Comment:
This forces _all_ callers of `localize()` and `trans()` to use an if-statement with the return value. `None` is not interchangeable with `str`. For instance, if we serialize a module param name of `null`, then our JS side will crash after printing a slew of proptype errors. And our test suite won't catch the crash.

Please `raise KeyError(module_id)` if the message does not exist. Callers that actually _want_ special behavior when the key doesn't exist may catch the exception.